docs/devcontainers/attach-container.md

```diff
@@ -94,10 +94,10 @@ To attach to a container in a Kubernetes cluster, first install the [Kubernetes
 
 > **Note:** Attached container configuration files are not yet supported for containers in a Kubernetes cluster.
 
-![Attach to Kubernetes Container](images/containers/k8s-attach.png)
+![Attach to Kubernetes Container](images/attach-container/k8s-attach.png)
 
 ## Next steps
 
-* [Create a Development Container](/docs/remote/create-dev-container.md) - Create a custom container for your work environment.
+* [Create a Dev Container](/docs/devcontainers/create-dev-container.md) - Create a custom container for your work environment.
 * [Advanced Containers](/remote/advancedcontainers/overview.md) - Find solutions to advanced container scenarios.
```

Comment:
I was envisioning we'd bring in the "Advanced Containers" node into the new Dev Containers node. Is that still the plan?

Reply:
I think we consider doing that but we'll want to investigate what that would look like after all the refactoring Burke did. We should also look at telemetry and ratings to see which advanced topics are the most popular and if we could cut some.

Reply:
Got it. I remembered we wanted to consider bringing some of the topics into the actual Dev Containers nodes and/or cutting or combining them. But I thought we were going to bring `Advanced Containers` into `Dev Containers` since we aren't leaving other container-related content behind in `Remote`.

Reply:
Oh, I get your question now. It's subtle, Advanced Container topics are under code.visualstudio.com/remote/advancedcontainers/*. This PR leaves then there but moves the "Advanced Containers" item, which has the redirect, from /docs/remote/containers-advanced.md to /docs/devcontainers/containers-advanced.

So in the new TOC, we'll still have the Advanced Containers item.

![image](https://user-images.githubusercontent.com/12818376/197891366-5d742ca7-4a20-4889-878a-a6df1e27f4a2.png)

Reply:
Oh perfect, thank you!